python/lsst/obs/hsc/makeHscRawVisitInfo.py

```diff
@@ -67,6 +64,13 @@ def setArgDict(self, md, argDict):
         argDict['era'] = self.eraFromLstAndLongitude(LST, self.observatory.getLongitude())
         argDict['darkTime'] = argDict['exposureTime']
 
+        # Rotation angle formula determined empirically from visual inspection
+        # of HSC images.  See DM-9111.
+        rotAngle = 270.0*degrees - self.popAngle(md, "INST-PA")
+        rotAngle.wrap()
```

Comment:
As of a few days ago `Angle.wrap` now returns a new angle rather than operating in place, so this code requires a minor change. The following should work, or you can break it into two lines if you prefer:
`rotAngle = (270.0*degrees - self.popAngle(md, "INST-PA")).wrap()`